compliance_checker/suite.py

```diff
@@ -35,8 +35,15 @@ def load_all_available_checkers(cls):
         for x in working_set.iter_entry_points('compliance_checker.suites'):
             try:
                 xl = x.load()
-
                 cls.checkers[xl.name] = xl
+                # set any specific version numbers or pass if we don't have
+                # the _supported_versions attribute
+                try:
+                    for ver in sorted(xl._supported_versions):
```

Comment:
Thinking it over, I'll probably refactor before submitting the PR.  It could get messy quick if we need to support many versions.